### PR TITLE
HS offered toast

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Functionality not hooked up; rendering on a Saved Search. 
For demo, to see the cool scroll action, use Tarek and make sure he has a bid for `2144` at the bottom of a long bid list.

image

### CODE REVIEW

File: src/Components/BidTracker/Messages/handshakeOffered.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const HandshakeOffered = ({ name, position }) => (
+  <span>
+    {name}, a handshake has been offered for position:
+    <Link to={position.link}>{position.name}</Link>
+    <div className="bottom-section">
+      Go to your Bid Tracker to accept the Handshake
+      <Link to="/profile/bidtracker">
```
Comment (mjoyce91) [inline src/Components/BidTracker/Messages/handshakeOffered.jsx:10]: We do also have functionality to pass an ID into this URL, which tells the browser to scroll to that bid/ID

Comment (elizabeth-jimenez) [reply src/Components/BidTracker/Messages/handshakeOffered.jsx:10]: woow cool. I'll add that!

File: src/Components/BidTracker/Messages/handshakeOffered.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        {/* <Link to="/profile/bidtracker/#bid-6_2441"> */}
+        {/* <Link to="/profile/bidtracker#bid-6_2441"> */}
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/Messages/handshakeOffered.jsx:12]: @mjoyce91  not working 🤷‍♀️

Comment (mjoyce91) [reply src/Components/BidTracker/Messages/handshakeOffered.jsx:12]: Hmm. Maybe using the wrong ID? I'll have to take a look

Comment (mjoyce91) [reply src/Components/BidTracker/Messages/handshakeOffered.jsx:12]: I don't think its the one with the `bid` prefix. Pretty sure its purely numerical

Comment (elizabeth-jimenez) [reply src/Components/BidTracker/Messages/handshakeOffered.jsx:12]: image
🤷‍♀️ 
just using id didn't work, either ⚰️

**Review by mjoyce91 — APPROVED** [review]

**Comment by elizabeth-jimenez** [comment]
@mjoyce91 Where's my "cool job"?

**Pull request merged**